assets.mjs

```diff
@@ -2,15 +2,62 @@
 // @ts-check
 
 import sharp from "npm:sharp@0.33.5";
+import { DOMParser } from "npm:linkedom@0.18.5";
+
+/** the text contained in the source SVG file */
+const text = await Deno.readTextFile("css.svg");
 
 // Names of the SVGs to use for generation
 const logoFileNames = ["css", "css.square", "css.dark", "css.light"];
 
 // Start the generation process for each logo
 for (const logoFileName of logoFileNames) {
+  const svg = new DOMParser().parseFromString(text, "image/svg+xml");
+
+  const fg = svg.getElementById("fg");
+  const bg = svg.getElementById("bg");
+  const title = svg.getElementById("css-logo-title");
+  const description = svg.getElementById("css-logo-description");
+
+  switch (logoFileName) {
+    case "css": {
+      // this is the source logo, do nothing!
+      break;
+    }
+    case "css.light": {
+      title.innerHTML = "CSS Logo Light";
+      description.innerHTML =
+        "A white square with rounded corners and the letters CSS inside in black";
+      fg.setAttribute("fill", "black");
+      bg.setAttribute("fill", "white");
+      break;
+    }
+    case "css.dark": {
+      title.innerHTML = "CSS Logo Dark";
+      description.innerHTML =
+        "A black square with rounded corners and the letters CSS inside in white";
+      fg.setAttribute("fill", "white");
+      bg.setAttribute("fill", "black");
+      break;
+    }
+    case "css.square": {
+      title.innerHTML = "CSS Logo Square";
+      description.innerHTML =
+        "A purple square with the letters CSS inside in white";
+      bg.setAttribute("d", "M0,0H1000V1000H-1000Z");
```

Comment:
The alternative would be to remove the `path` element and replace it with a `rect` as is currently the case, but it would save very few bytes and would be much more complex.

Instead we draw a full-width square by:
1. `M0,0` moving to the origin point
2. `H1000` drawing a 1000 horizontal line right
3. `V1000` drawing a 1000 vertical line down
4. `H1000` drawing a 1000 horizontal line left
5. `Z` closing the rectangle